src/main/java/WayofTime/bloodmagic/item/ItemSacrificialDagger.java

```diff
@@ -37,6 +34,8 @@
 
 public class ItemSacrificialDagger extends ItemEnum<ItemSacrificialDagger.DaggerType> implements IMeshProvider {
 
+    private boolean hasMaxIncense;
```

Comment:
Nope not how this works.

Reply:
:face_palm:

Reply:
Well, how DOES it work then?
I take it you mean: "This is not how/where we declare booleans."
Where should I put it?

Reply:
no, not static. As items are flyweights, making the value static does not change anything, as instance variables are effectively static.

Reply:
It's great that you all tell me what I shouldn't do but what should I do instead of that what I shouldn't do?

Because simply saying what should not be done doesn't change the fact that I didn't know better in that case and probably didn't find it out on my own while I was doing what I ended up doing.